docs/Installing.asciidoc

```diff
@@ -697,6 +697,24 @@ This will allow the workers to download the assets from the web UI and use them
 locally. If `TESTPOOLSERVER` is set tests and needles will also be cached by the
 worker.
 
+=== Enable linking to setting files
+
+You can configure openQA to display the context of specific files in the Settings.
+There are files which the test suite is using and you want to investigate from the
+OpenQA UI directly.
+To enable particular settings to present as a link in the Settings we can setup the
+underlined directories in `/etc/openqa/openqa.ini`.
+
+[source,ini]
+--------------------------------------------------------------------------------
+# Configuration for Setting UI
+[job_settings_ui]
+keys_to_render_as_links=schedule,autoyast_sle15
```

Comment:
i am not sure i get this. What is wrong with it? Can you give me an example?

Reply:
In my opinion the config file should look like this:

```
[job_settings_ui]
keys_to_render_as_links=YAML_SCHEDULE,AUTOYAST
```

So you specify the *keys* of the settings which values should be rendered as links. For me this makes much more sense than applying the rendering blindly to any setting which value matches a certain pattern.

Of course your code needs to be changes so it would actually work like this. (I suggest to create an app-global hash containing all configured keys after parsing the configuration. This hash can be exposed to the template as stash variable and should allow making the distinction very fast.)